root/etc/cont-init.d/50-config

```diff
@@ -35,11 +35,19 @@ done
 
 # make our folders and links
 mkdir -p \
-	/config/{log/letsencrypt,log/fail2ban,etc/letsencrypt,fail2ban,crontabs,dns-conf,geoip2db} \
+	/config/{log/letsencrypt,log/fail2ban,etc/letsencrypt,fail2ban,crontabs,dns-conf,geoip2db,renewal-hooks} \
 	/var/run/fail2ban
 rm -rf /etc/letsencrypt
 ln -s /config/etc/letsencrypt /etc/letsencrypt
 
+rm -rf /config/etc/letsencrypt/renewal-hooks
+ln -s ../../renewal-hooks /config/etc/letsencrypt/renewal-hooks
+
+# copy default renewal hooks
+chmod -R +x /defaults/renewal-hooks
```

Comment:
chmod is only on the default hooks at `/defaults/renewal-hooks` not on the copied `/config/renewal-hooks/` in order to not change behaviour when docker is restarted.